index.md

```diff
@@ -8,7 +8,7 @@ lang: en
 
 Lessons about the Elixir programming language, inspired by Twitter's [Scala School](http://twitter.github.io/scala_school/).
 
-Available in [Việt ngữ][vi], [汉语][cn], [Español][es], [Slovenčina][sk], [日本語][jp], [Polski][pl], [Português][pt], [Русском][ru], [Bahasa Melayu][my], [Українською][uk], [한국어][ko], [L'italiano][it], [Deutsch](de), [বাংলা](bn) and other.
+Available in [Việt ngữ][vi], [汉语][cn], [Español][es], [Slovenčina][sk], [日本語][jp], [Polski][pl], [Português][pt], [Русском][ru], [Bahasa Melayu][my], [Українською][uk], [한국어][ko], [L'italiano][it], [Deutsch](de), [বাংলা](bn), [Türkçe](tr) and other.
```

Comment:
It looks like the conflict is on this line.  I believe it should be something akin to this:

```markdown
Available in [Việt ngữ][vi], [汉语][cn], [Español][es], [Slovenčina][sk], [日本語][jp], [Polski][pl], [Português][pt], [Русском][ru], [Bahasa Indonesia][id], [Bahasa Melayu][my], [Українською][uk], [한국어][ko], [L'italiano][it], [Deutsch](de), [বাংলা](bn), [Türkçe](tr) and others.
```

You can try to resolve the conflict by rebasing your branch on origin and fixing any conflicts, to do that try:

```elixir
cd elixir-school
git remote add origin git@github.com:doomspork/elixir-school.git
git fetch --all
git rebase origin/master
```

The above commands should notify you of a conflict in the rebase.  Once you fix `en/index.md` you'll need to do:

```elixir
git add en/index.md
git rebase --continue
```

I hope that helps!